src/main/java/bro/command/ExitCommand.java

```diff
@@ -0,0 +1,19 @@
+package bro.command;
+
+import bro.Storage;
+import bro.TaskList;
+import bro.Ui;
+
+public class ExitCommand extends Command {
+
+    public ExitCommand(){}
```

Comment:
Perhaps you might want to include a whitespace between the bracket and the brace?